vanilla-framework/vanilla-framework
# Update docs/image-position page content

## Done
- Updated copy 👍 
- Updated demo example to showcase live example 😉 
- Check content matches copy doc [here](https://docs.google.com/document/d/1BaY3k01-6zvhC67PMxZD44IKHs4_pTpPm_ix98rHW98/edit#heading=h.7ykvnb39dnrl)

## QA
- Pull code
- Run /docs > ./run
  - Check http://0.0.0.0:8104/en/utilities/image-position
- Pull code
- Run ./run
- Review updated examples
  - Check http://0.0.0.0:8101/vanilla-framework/examples/utilities/image-position/bottom-left/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/utilities/image-position/bottom-right/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/utilities/image-position/bottom/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/utilities/image-position/left/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/utilities/image-position/right/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/utilities/image-position/top-left/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/utilities/image-position/top-right/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/utilities/image-position/top/

## Screenshot
![Screenshot 2019-03-27 at 14 57 58](https://user-images.githubusercontent.com/17748020/55086711-c2850e00-50a0-11e9-8204-6bdfda062224.png)
